### PR TITLE
Created the base homepage

### PR DESCRIPTION
**Pull request by kdmontero**
Added register and login buttons which are linked to newly created (placeholder) register and login page

### CODE REVIEW

File: src/routes/index.tsx [hunk 1]
```diff
@@ -4,13 +4,15 @@ import {
   Route,
 } from "react-router-dom";
 
-import { Account, Home, Jebaited, RoomDetail, RoomList } from "../pages";
+import { Account, Home, Register, Login, Jebaited, RoomDetail, RoomList } from "../pages";
 
 // You can do this:
```
Comment (ksemilla) [inline src/routes/index.tsx]: remove comment

File: src/routes/index.tsx [hunk 2]
```diff
@@ -4,13 +4,15 @@ import {
...
 export const router = createBrowserRouter(
   createRoutesFromElements(
     <Route path="/">
       <Route path="" element={<Home />} />
+      <Route path="register" element={<Register />} />
+      <Route path="login" element={<Login />} />
       <Route path="my-account" element={<Account />} />
       <Route path="rooms" element={<Jebaited />}>
```
Comment (ksemilla) [inline src/routes/index.tsx]: replace name "Jebaited"

File: src/pages/home/index.tsx [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
 const Home = () => {
-  return <div className="text-3xl">this is home page</div>;
+  return <>
+    <div className="bg-slate-400 h-window">
+      <div className="p-2 bg-slate-500 inline-block">
+        <h1 className="text-4xl">BUNOTAN</h1>
+      </div>
+      <div className="space-y-5 flex flex-col h-screen justify-center items-center">
+        <Link to="/register">
+          <button className="py-2 px-6 bg-blue-500 inline-block text-3xl">
```
Comment (ksemilla) [inline src/pages/home/index.tsx:12]: Put className in Link component ex.`<Link to="/" className="flex">`

File: src/pages/home/index.tsx [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
+            Register
+          </button>
+        </Link>
+        <Link to="/login">
```
Comment (ksemilla) [inline src/pages/home/index.tsx]: Do the same as register

File: src/routes/index.tsx (hunk 1 above)
Comment (kdmontero) [reply src/routes/index.tsx]: removed

File: src/routes/index.tsx (hunk 2 above)
Comment (kdmontero) [reply src/routes/index.tsx]: replaced with MyRooms

File: src/pages/home/index.tsx (hunk 4 above)
Comment (kdmontero) [reply src/pages/home/index.tsx]: added className

File: src/pages/home/index.tsx (hunk 3 above)
Comment (kdmontero) [reply src/pages/home/index.tsx:12]: added className

File: src/pages/home/index.tsx [hunk 5]
```diff
@@ -1,5 +1,26 @@
...
+
 const Home = () => {
-  return <div className="text-3xl">this is home page</div>;
+  return <>
+    <div className="bg-slate-400 h-window">
+      <div className="p-2 bg-slate-500 inline-block">
+        <h1 className="text-4xl">BUNOTAN</h1>
+      </div>
+      <div className="space-y-5 flex flex-col h-screen justify-center items-center">
+        <Link to="/register", className="flex">
```
Comment (ksemilla) [inline src/pages/home/index.tsx]: Remove comma

File: src/pages/home/index.tsx [hunk 6]
```diff
@@ -1,5 +1,26 @@
...
+          <button className="py-2 px-6 bg-blue-500 inline-block text-3xl">
+            Register
+          </button>
+        </Link>
+        <Link to="/login", className="flex">
```
Comment (ksemilla) [inline src/pages/home/index.tsx]: Remove comma

File: src/pages/room/index.ts [hunk 7]
```diff
@@ -1,5 +1,5 @@
 import RoomList from "./list";
 import RoomDetail from "./detail";
-import Jebaited from "./jebaited";
+import MyRooms from "./myrooms";
```
Comment (ksemilla) [inline src/pages/room/index.ts:3]: `./myrooms` directory doesn't exist. Please include in the commit.

File: src/pages/home/index.tsx (hunk 5 above)
Comment (kdmontero) [reply src/pages/home/index.tsx]: removed

File: src/pages/home/index.tsx (hunk 6 above)
Comment (kdmontero) [reply src/pages/home/index.tsx]: removed

File: src/pages/room/index.ts (hunk 7 above)
Comment (kdmontero) [reply src/pages/room/index.ts:3]: renamed jebaited to myrooms

**Pull request merged**